programmers/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,17 @@
+function solution(numbers) {
+    
+    const NUM_LEN = numbers.length;
+    let answer = [];
+    let sortAnswer = [];
+    let sum = 0;
+    
+    for (let i = 0; i < NUM_LEN; i++) {
+        for (let j = i + 1; j < NUM_LEN; j++) {
+            sum = numbers[i] + numbers[j];
+            
+            if (answer.indexOf(sum) === -1) answer.push(sum);
```

Comment:
```suggestion
            if (!answer.includes(sum)) answer.push(sum);
```
indexOf도 있지만 includes는 있으면 true 없으면 false를 주기때문에 비교할때는 includes를 많이 사용해요

false나 true같은 boolean값과 비교할때는
!를 하면 false와 같다고 비교
result.includes(add)같이 그냥 쓰게 되면 true랑 비교하게 됩니다.